pg_metadata/install/sql/upgrade/upgrade_to_1.1.0.sql

```diff
@@ -137,4 +137,92 @@ COMMENT ON FUNCTION pgmetadata.update_table_comment_from_dataset() IS 'Update th
+('dataset.categories', 'BOU', 'Grenzen', 'gesetzlich festgelegte Grenzen'),
+('dataset.categories', 'STR', 'Bauwerke', 'anthropogene Bauten'),
+('dataset.categories', 'TRA', 'Verkehrswesen', 'Mittel und Wege zur Beförderung von Personen und/oder Gütern'),
+('dataset.categories', 'INT', 'Militär und Aufklärung', 'Militärbasen, militärische Einrichtungen und Aktivitäten'),
+('dataset.categories', 'LOC', 'Ortsangaben', 'Positionierungsangaben und -dienste'),
+('dataset.categories', 'CLI', 'Atmosphäre', 'Prozesse und Naturereignisse der Atmosphäre inkl. Klimatologie und Meteorologie'),
+('dataset.categories', 'FAR', 'Landwirtschaft', 'Tierzucht und/oder Pflanzenanbau'),
+('dataset.categories', 'ENV', 'Umwelt', 'Umweltressourcen, Umweltschutz und Umwelterhaltung'),
+('dataset.categories', 'OCE', 'Meere', 'Merkmale und Charakteristika von salzhaltigen  Gewässern (außer Binnengewässern)'),
+('dataset.categories', 'BIO', 'Biologie', 'Flora und/oder Fauna in der natürlichen Umgebung'),
+('dataset.categories', 'IMA', 'Oberflächenbeschreibung', 'Basiskarten und -daten'),
+('dataset.categories', 'SOC', 'Gesellschaft', 'kulturelle und gesellschaftliche Merkmale'),
+('dataset.categories', 'ECO', 'Wirtschaft', 'wirtschaftliche Aktivitäten, Verhältnisse und Beschäftigung'),
+('dataset.categories', 'UTI', 'Ver- und Entsorgung, Kommunikation', 'Energie-, Wasser- und Abfallsysteme, Kommunikationsinfrastruktur und -dienste'),
+('dataset.confidentiality', 'OPE', 'offen', 'Keine Einschränkungen des Zugriffs auf diese Daten'),
+('dataset.confidentiality', 'RES', 'eingeschränkt', 'Der Zugriff auf die Daten ist auf ausgewählte Nutzer beschränkt'),
+('dataset.publication_frequency', 'DAY', 'täglich', 'Daten werden täglich aktualisiert'),
+('dataset.publication_frequency', 'MON', 'monatlich',  'Daten werden monatlich aktualisiert'),
+('dataset.publication_frequency', 'YEA', 'jährlich', 'Daten werden jährlich aktualisiert'),
+('dataset.publication_frequency', 'NEC', 'bei Bedarf',  'Daten werden bei Bedarf aktualisiert'),
+('dataset.publication_frequency', 'WEE', 'wöchentlich', 'Daten werden wöchentlich aktualisiert')
+ON CONFLICT DO NOTHING;
+
+UPDATE pgmetadata.glossary AS g
+SET (label_de, description_de)
+= (t.label_de, t.description_de)
+FROM pgmetadata.t_glossary AS t
+WHERE g.field = t.field AND g.code = t.code;
+
+DROP TABLE pgmetadata.t_glossary;
+
+
+-- new terms
+
+INSERT INTO pgmetadata.glossary (id, field, code, label_en, description_en, item_order, label_fr, description_fr, label_it, description_it, label_es, description_es, label_de, description_de) VALUES
+(130, 'dataset.license', 'dl-de/by-2-0', 'Data licence Germany – attribution – version 2.0', NULL, 80, NULL, NULL, NULL, NULL, NULL, NULL, 'Datenlizenz Deutschland – Namensnennung – Version 2.0', NULL),
+(131, 'dataset.license', 'proj', 'restricted use for project-related work', NULL, 90, NULL, NULL, NULL, NULL, NULL, NULL, 'nur für Projektbearbeitung', NULL)
```

Comment:
```suggestion
(131, 'dataset.license', 'proj', 'Restricted use for project-related work', NULL, 90, NULL, NULL, NULL, NULL, NULL, NULL, 'nur für Projektbearbeitung', NULL)
```

I will let you manage capital letters in German, but I think in English, we can ;-)

Reply:
Yes, thanks for the correction of the Englisch version